<launch>
  <node name="px4_controller" pkg="url_uavugv" type="rect_uav.py" output="screen">
    <param name="robot" type="str" value="" />
    <param name="length" type="double" value="10.0" />
    <param name="target_altitude" type="double" value="3.0" />
    <param name="max_speed" type="double" value="0.5" />
    <param name="laps_completed" type="int" value="1" />
    <param name="use_gps" type="bool" value="True" />
  </node>
</launch>

